### ISSUE TITLE
dict.get does not type check correctly when keys are tuples

### ISSUE THREAD

**Issue by shamrin** [microsoft/pyright#252]
**Describe the bug**
Type checking `dict.get` does not work when keys are tuples.

**To Reproduce**
```python
from typing import Dict, Tuple
from typing_extensions import Literal

Language = Literal['en', 'de']

translations: Dict[Tuple[Language, str], str] = {}
translations['en', 'one'] = 'one'
translations['de', 'one'] = 'eins'

def translate1(language: Language, code: str) -> str:
    return translations.get((language, code), code)

def translate2(language: Language, code: str) -> str:
    return translations[(language, code)]
```

**Expected behavior**
I would expect `translate1` not produce an error.

**Screenshots or Code**
<img width="554" alt="image" src="https://user-images.githubusercontent.com/510678/65026144-a459e580-d940-11e9-887d-7201038cebd2.png">

**VS Code extension or command-line**
Name: Pyright
Id: ms-pyright.pyright
Description: VS Code static type checking for Python
Version: 1.0.62
Publisher: ms-pyright
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=ms-pyright.pyright

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next published version of pyright.

**Comment by erictraut** [comment]
This is now fixed in pyright 1.0.63, which I just published.
